src/extension.ts

```diff
@@ -304,8 +326,11 @@ function allMatches(regex: RegExp, string: string, group: number) {
     }
 }
 
-function replaceContent(editor: TextEditor, content: string) {
+function replaceContent(editor: TextEditor, content: string, documentLanguage: string = '') {
     const document = editor.document;
+    if (documentLanguage) {
```

Comment:
It would be better to always update because in the case where the document is reused, it might receive JSON again after plain text.